src/Visualization/Python/PlotCce.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+from typing import Optional, Sequence
+
+import click
+import h5py
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from spectre.Visualization.Plot import (
+    apply_stylesheet_command,
+    show_or_save_plot_command,
+)
+from spectre.Visualization.ReadH5 import available_subfiles, to_dataframe
+
+logger = logging.getLogger(__name__)
+
+
+def _parse_modes(ctx, param, all_modes):
+    result = []
+    for mode in all_modes:
+        l_and_m = mode.split(",")
+        if len(l_and_m) != 2:
+            raise click.BadParameter(
+                f"The mode {mode} must be specified as 'l,m'"
+            )
+        result.append(f"Real Y_{mode}")
+        result.append(f"Imag Y_{mode}")
+    return result
+
+
+@click.command(name="cce")
+@click.argument(
+    "h5_filename",
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+    nargs=1,
+)
+@click.option(
+    "--modes",
+    "-m",
+    multiple=True,
+    callback=_parse_modes,
+    required=True,
+    help=(
+        "Which mode to plot. Specified as 'l,m'. Will plot both real and"
```

Comment:
I personally don't like using `nargs=2` for click.options (for click.arguments they are good). It feels more natural to me to have comma separated value (maybe it's because there isn't a space so it feels like one option rather than two).

Reply:
We're currently not consistent with comma vs space, so either is fine with me